tests/test_api_path_traversal.py

```
@@ -0,0 +1,157 @@
+            assert e.status_code == 400
+
+        # Test bad artifact_name
+        try:
+            await get_wisdom_content("valid_run", "../bad_artifact")
+            pytest.fail("Should have raised HTTPException(400)")
+        except HTTPException as e:
+            assert e.status_code == 400
+
+    run_async(run_test())
+
+def test_wisdom_apply_patch_traversal(mock_app_context):
```

Ruff (0.15.1)

[warning] Unused function argument: `mock_app_context`

(ARG001)